cyclus/system.py

```diff
@@ -12,7 +12,7 @@ def PY_VERSION_TUPLE():
 
 @lazyobject
 def curio():
```

Comment:
Aren't these functions supposed to return an instance of the module they are importing?